README.md

```diff
@@ -124,10 +124,28 @@ and remove the container when the application exits (`--rm`). After installing a
 runs `scripts/start.sh`
 which first starts MongoDB server in a separate thread and then runs `live_main.py`.
 
-_Note: Similarly to requiring to requiring the additional flag of `--platform linux/amd64` to build the Docker container with an arm64 machine, running said container also requires the same flag.
+_Note: Similar to requiring the additional flag of `--platform linux/amd64` to build the Docker container with an arm64 machine, running said container also requires the same flag.
 :warning: **The performance of the container under this emulation will be incredibly poor. If possible, use a x86_64/amd64 system**._
 
-#### ssh into the container
+#### Run with file input
+
+To avoid typing the input every time, you can provide an input file and get output in an output file. We need to mount
+the directory containing the data.
+
+```commandline
+docker build -t macaw . && docker run --rm -i --name=macaw_test_container \
+-v /Users/amitgh/PycharmProjects/DrQA/data:/usr/src/app/DrQA/data \
+-v $("pwd")/stanford-corenlp-full-2017-06-09:/usr/src/app/stanford-corenlp-full-2017-06-09 \
+-v /Users/amitgh/PycharmProjects/Maruna/macaw/data:/usr/src/app/data \

```

Comment:
It can be. I kept it complete path just to show that both types of paths can be given. But I guess it's understood. So I've made it `$("pwd")/data` now.